packages/bsky/src/db/migrations/20231003T202833377Z-create-moderation-subject-status.ts

```diff
@@ -0,0 +1,72 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_action')
+    .renameColumn('reason', 'comment')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .alterColumn('comment')
+    .dropNotNull()
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('refEventId', 'integer')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('meta', 'jsonb')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .renameTo('moderation_event')
+    .execute()
+  await db.schema
+    .createTable('moderation_subject_status')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+
+    // Identifiers
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    // Default to '' so that we can apply unique constraints on did and recordPath columns
+    .addColumn('recordPath', 'varchar', (col) => col.notNull().defaultTo(''))
+    .addColumn('recordCid', 'varchar')
+
+    // human review team state
+    .addColumn('reviewState', 'varchar', (col) => col.notNull())
+    .addColumn('note', 'varchar')
+    .addColumn('muteUntil', 'varchar')
+    .addColumn('lastReviewedAt', 'varchar')
+
+    // report state
+    .addColumn('lastReportedAt', 'varchar')
+
+    // visibility/intervention state
+    .addColumn('takendown', 'boolean', (col) => col.defaultTo(false).notNull())
+    .addColumn('suspendUntil', 'varchar')
+
+    // timestamps
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('updatedAt', 'varchar', (col) => col.notNull())
+    .addUniqueConstraint('did_record_path_unique_idx', ['did', 'recordPath'])
```

Comment:
Since constraints all share a namespace in postgres, I think we've been prefixing them with the table name.  You'll find some examples if you search for `addUniqueConstraint` in the codebase.

Reply:
I vaguely remember there was a length limit on constraint keys so probably was subconsciously trying to keep it brief. Set a new name with the table prefix, seems fine :)